src/components/UI/Form/EmojiInput/EmojiInput.tsx

```diff
@@ -83,111 +28,31 @@ export const EmojiInput = ({
 }: EmojiInputProps) => {
   const [showEmojiPicker, setShowEmojiPicker] = useState(false);
 
-  const updateValue = (input: any, isEmoji = false) => {
-    const editorContentState = value.getCurrentContent();
-    const editorSelectionState: any = value.getSelection();
-    const ModifiedContent = Modifier.replaceText(
-      editorContentState,
-      editorSelectionState,
-      isEmoji ? input.native : input
-    );
-    let updatedEditorState = EditorState.push(value, ModifiedContent, 'insert-characters');
-    if (!isEmoji) {
-      const editorSelectionStateMod = updatedEditorState.getSelection();
-      const updatedSelection = editorSelectionStateMod.merge({
-        anchorOffset: editorSelectionStateMod.getAnchorOffset() - 1,
-        focusOffset: editorSelectionStateMod.getFocusOffset() - 1,
-      });
-      updatedEditorState = EditorState.forceSelection(updatedEditorState, updatedSelection);
-    }
-    props.form.setFieldValue(name, updatedEditorState);
-  };
-
-  const handleKeyCommand = (command: any, editorState: any) => {
-    if (command === 'underline') {
-      return 'handled';
-    }
-    if (command === 'bold') {
-      updateValue('**');
-    } else if (command === 'italic') {
-      updateValue('__');
-    } else {
-      const newState = RichUtils.handleKeyCommand(editorState, command);
-      if (newState) {
-        props.form.setFieldValue(name, newState);
-        return 'handled';
-      }
-    }
-    return 'not-handled';
-  };
-
-  const draftJsChange = (editorState: any) => {
+  const lexicalChange = (editorState: any) => {
```

Comment:
```suggestion
  const lexicalChange = (editorState: string) => {
```
Let's add types where we know